sdk/tests/conformance/extensions/webgl-compressed-texture-etc.html

```diff
@@ -140,11 +140,12 @@
 
     debug("");
     if (contextVersion >= 2) {
+      var expectedError = isPositive ? gl.INVALID_OPERATION: [gl.INVALID_ENUM, gl.INVALID_OPERATION];
```

Comment:
We don't know which error will be raised early by implementations, either INVALID_ENUM or INVALID_OPERATION is possible in this case.